rubrikcdm/resource_rubrik_assign_sla_test.go

```diff
@@ -0,0 +1,85 @@
+package rubrikcdm
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/rubrikinc/rubrik-sdk-for-go/rubrikcdm"
+)
+
+func TestAccRubrikAssignSla_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccCheckEnvVariables(t, []string{"RUBRIK_CDM_TEST_VM", "RUBRIK_CDM_TEST_SLA"})
+			testAccCheckRubrikAssignSLAPreCheck(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckRubrikAssignSLADestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRubrikAssignSLAConfig(os.Getenv("RUBRIK_CDM_TEST_VM"), os.Getenv("RUBRIK_CDM_TEST_SLA")),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("rubrik_assign_sla.assign_sla", "object_name", os.Getenv("RUBRIK_CDM_TEST_VM")),
+					resource.TestCheckResourceAttr("rubrik_assign_sla.assign_sla", "sla_name", os.Getenv("RUBRIK_CDM_TEST_SLA")),
+				),
+			},
+		},
+	})
+}
+
+func testAccRubrikAssignSLAConfig(vmName string, slaName string) string {
+	return fmt.Sprintf(`
+resource "rubrik_assign_sla" "assign_sla" {
+	object_name = "%s"
+	object_type = "vmware"
+	sla_name    = "%s"
+	timeout     = 15
+	}
+`, vmName, slaName)
+}
```

Comment:
Formatting looks off here

Reply:
Agreed, but that matches the formatting used in all of the other Terraform acceptance tests I looked at, so I didn't attempt to format it in any other way. Ex: https://github.com/terraform-providers/terraform-provider-gitlab/blob/563161f80c7ada085fffef74efe1ddf84dd9254d/gitlab/resource_gitlab_user_test.go#L155. Other tests I've read through follow this same formatting.

Reply:
#shipit